include/picongpu/simulation_defines/param/speciesAttributes.param

```diff
@@ -254,4 +254,15 @@ namespace picongpu
      */
     alias( exchangeMemCfg );
 
+    /** alias to specify the boundary condition for particles
+     *
+     * The default behavior if this alias is not given to a species is that the
+     * particles which leave the global simulation box were deleted.
```

Comment:
where

Reply:
Add sentence: `This also notifies all plugins that can handle leaving particles.`